splash/tests/test_execute.py

```diff
@@ -3653,3 +3653,34 @@ def test_release_with_bad_arguments(self):
 
         msg = "coordinate must be a number"
         self.assertScriptError(resp, ScriptError.SPLASH_LUA_ERROR, msg)
+
+
+class KeyEventsTest(BaseLuaRenderTest):
+    def test_send_raw_key_events(self):
+        resp = self.request_lua("""
+             function main(splash)
+                assert(splash:go(splash.args.url))
+                join_inputs = splash:jsfunc([[
+                    function () {
+                        var inputs = document.getElementsByTagName('input');
+                        var values = [];
+                        for (var i = 0; i < inputs.length; i++) {
+                            values.push(inputs[i].value);
+                        }
+                        return values.join('|');
+                    }
+                ]])
+                splash:send_keys('', 'Tab')
```

Comment:
@kmike So, is the proposed API better than before? `send_key` + `send_keys` might be confusing.